crates/cxx-qt-gen/src/generator/cpp/mod.rs

```diff
@@ -73,6 +73,55 @@ impl GeneratedCppBlocks {
     }
 }
 
+mod utils {
```

Comment:
any reason for this not being in a `cpp/utils.rs` ?

Reply:
Just thought it's a bit too small to deserve it's own file, but don't mind either way :)
Will move into a file :+1:

Reply:
Done